test/range_test.py

```diff
@@ -509,11 +509,13 @@ def output(self):
                         'spl': MockFile(base + '/something.spl'),
                     }
 
-        with self.assertRaises(NotImplementedError):
+        def test_raise_not_implemented():
```

Comment:
In theory, I agree. However some tests fail when importing helpers even when helpers is an empty file (https://github.com/spotify/luigi/pull/760#issuecomment-75191878). It can be fixed by adding a `__init__.py` file in the test folder but that makes fail other tests.